cuba.yml

```diff
@@ -545,3 +545,23 @@ CUBA_KEYS:
     definition: Classification of a fluid flow type
     shape: [1]
     type: integer
+
+  VECTOR:
+    definition: A vector
+    shape: [1]
+    type: double
```

Comment:
> CUBA.DIRECTION is defined in cuba.yml, is that what you want?
direction is something else, it can be defines by a vector and origin system, or it may have other keys, such as up down, left right! we should ask those who defined it to elaborate it. 
a vector here is the mathematical basic definition. We need a better definition of direction and to see where it is used!